_im-plugin/notifications-settings.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: Configuring notifications settings
+parent: Managing indexes
+nav_order: 100
+---
+
+## Configuring notifications settings
+
+Introduced 2.8
+{: .label .label-purple }
+
+Notifications settings allow users to control how they receive notifications about index events. You can use either OpenSearch Dashboards or the REST API to configure notifications. Supported communication channels include Amazon Chime, Amazon Simple Notification Service (Amazon SNS), Amazon Simple Email Service (Amazon SES), email through SMTP, Slack, and custom webhooks. Supported index operations include resize, reindex, open, and force merge. See [Notifications](https://opensearch.org/docs/latest/observing-your-data/notifications/index/) for further details. 
+
+To configure configure the notifications settings for long-running index operations using OpenSearch Dashboards, go to **Dev Tools** under the main menu. When creating notifications using the API, the LRON setting can be configured two ways: as a concrete task or as a global and persistent task. The LRON setting with `task_id` is ad hoc and automatically deletes when the task ends. The LRON setting using `action_name` is global and persists and applies to all operations of this action type.
+
+This documentation is intended for IT administrators as a guide to configuring notifications settings.
+
+## Create, read, update, and delete (CRUD) operations
+
+The following code examples show how to configure notifications settings for long-running index operations.
+
+### Create 
+
+The following example request creates the notification using the `channels` field:
+
+```json
+POST /_plugins/_im/lron
+{
+  "lron_config": {
+      "task_id":"dQlcQ0hQS2mwF-AQ7icCMw:12354",
+      "action_name":"indices:data/write/reindex",
+      "lron_condition": {
+        "success": false,
+        "failure": true
+      },
+      "channels":[
+          {"id":"channel1"},
+          {"id":"channel2"}
+      ]
+  }
+}
+```
+
+which results in the following response: 
+
+```json
+{
+  "_id": "LRON:dQlcQ0hQS2mwF-AQ7icCMw:12354",
+  "lron_config": {
+    "lron_condition": {
+      "success": false,
+      "failure": true
+    },
+    "task_id": "dQlcQ0hQS2mwF-AQ7icCMw:12354",
+    "action_name": "indices:data/write/reindex",
+    "channels": [
+      {
+        "id": "channel1"
+      },
+      {
+        "id": "channel2"
+      }
+    ]
+  }
+}
+```
+
+## Read
+
+The following GET request examples retrieve the current notifications settings for a user or group. This information can be used to determine what types of notifications the user or group is currently recieveing, and to make changes to the notifications settings as needed. 
+
+
+Get one document
+
+```bash
+ GET /_plugins/_im/lron/{lronID}
```

Comment:
@zhichao-aws Is the <lronID> the `_id` from the response when you create an lron setting?

Reply:
Global: include copy-curl for all requests and make them json instead of bash.

Reply:
Yes, it is the `_id` from the response. For global lron_config, it will be "LRON:"+action_name. For one-time lron_config, it will be "LRON:"+task_id.
Here is one thing I want to call out, there maybe char "/" in action_name, and it may be in `_id`. If users are using dev tools, they will need to use [http encode](https://www.w3schools.com/tags/ref_urlencode.ASP) to transfer "/" to "%2F" in action name. e.g. PUT _plugins/_im/lron/LRON:indices:admin%2Fopen in dev tools